README.rst

```diff
@@ -133,15 +133,22 @@ More examples
 
 .. code-block:: python
 
-    >>> import boto3
+    >>> import os, sys, boto3
```

Comment:
also, I thought that this is a good idea to get rid `repr` function that used for strings (we don't really need them), see my "pathlib-patching" example